api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/PersonAttributeDataConverter.java

```diff
@@ -24,8 +26,26 @@ public Object convert(Object obj) {
         if (obj == null) {
             return "";
         }
-
-        return ((PersonAttribute)obj).getValue();
+        String personAttribute = ((PersonAttribute)obj).getValue();
+        if(personAttribute!=null)
```

Comment:
@solemabrothers as good coding practice, check for null first then return

Reply:
Done